spec/controllers/links_controller_spec.rb

```diff
@@ -41,47 +41,149 @@
     end
   end
 
-  describe '#update' do
-    it 'updates link and redirects to HTTP_REFERER' do
-      link = Link.create(given_url: Faker::Internet.url,
-                         slug: Faker::Internet.slug)
-      post :update, id: link.id,
-                    slug: 'some',
-                    given_url: link.given_url,
-                    active: true
-      expect(response.status).to eq 302
-      expect(flash[:notice]).to be_present
+  describe 'GET #show' do
+    let(:user) { create(:user) }
+    let(:link) { create(:link) }
+
+    context 'when user is signed in' do
+      before do
+        user.links << link
+        session[:id] = user.id
+        get :show, link_id: link.id
+      end
+
+      it 'displays the dashboard' do
+        expect(response).to render_template 'show'
+      end
+
+      it 'has an array containing my links' do
```

Comment:
**Critical - Test Documentation**
The description here does not necessarily match the expectations in this example.

With such description, I'll expect an assertion of the type of data as well as whether links exist in that array.